docs/user-guide/supported-protocols/postgresql.md

```diff
@@ -1,5 +1,14 @@
 # PostgreSQL
 
+PostgreSQL wire protocol is supported by GreptimeDB, which means you can use
+standard PostgreSQL client to connect to DB instances and execute
+queries. Note that when writing SQL queries, GreptimeDB follows MySQL dialect
+instead of PostgreSQL.
+
+Both simple and extended query subprotocols are supported, except for those
```

Comment:
```suggestion
GreptimeDB supports simple and extendable query subprotocols, except parameter inference 
```

Reply:
here extended query is a name for subprotocol. but let reorganise later part of the sentence.